app/src/lib/stores/app-store.ts

```diff
@@ -6376,25 +6376,6 @@ export class AppStore extends TypedBaseStore<IAppState> {
     targetBranch: Branch,
     commits: ReadonlyArray<Commit>
   ): void {
-    this.repositoryStateCache.updateMultiCommitOperationState(
```

Comment:
Have to refactor to use initializer method - done in a different PR; just removing for now.